dev/create-release/release-tag.sh

```diff
@@ -24,7 +24,7 @@ function exit_with_usage {
   local NAME=$(basename $0)
   cat << EOF
 usage: $NAME
-Tags a Spark release on a particular branch.
+Tags a Spark release on a particular branch. Must push after
```

Comment:
"Must push after" - it reads a bit weird, is it unfinished sentence?

Reply:
I'll reword to "You must push the tags after."

Reply:
thanks. looks good.